assets/installer/manifest.yaml

```diff
@@ -0,0 +1,220 @@
+# epinio-compose.yaml
+components:
+  - id: linkerd
+    type: yaml
+    source:
+      path: assets/installer/linkerd-job.yaml
+    waitComplete:
+      - type: "job"
+        selector: "linkerd-install"
+        namespace: linkerd
+
+  - id: traefik
+    needs: linkerd
+    namespace: traefik
+    type: helm
+    source:
+      name: traefik
+      url: https://helm.traefik.io/traefik/traefik-10.3.4.tgz
+    waitComplete:
+      - type: "pod"
+        selector: "app.kubernetes.io/name=traefik"
+      - type: "loadbalancer"
+        selector: "traefik"
+    values:
+      - name: globalArguments
+        value: ""
+      - name: "deployment.podAnnotations.linkerd\\.io/inject"
+        value: "enabled"
+      - name: "ports.web.redirectTo"
+        value: "websecure"
+      - name: "ingressClass.enabled"
+        value: "true"
+      - name: "ingressClass.isDefaultClass"
+        value: "true"
+      # - name: "service.spec.loadBalancerIP"
+      #   value: "{{ .Values.loadbalancerIP }}"
+
+  - id: kubed
+    namespace: kubed
+    type: helm
+    source:
+      name: kubed
+      path: assets/installer/kubed-v0.12.0.tgz
+
+  - id: cert-manager
+    namespace: cert-manager
+    type: helm
+    source:
+      name: cert-manager
+      path: assets/installer/cert-manager-v1.5.4.tgz
+    values:
+      - name: "installCRDs"
+        value: "true"
+      - name: "extraArgs[0]"
+        value: "--enable-certificate-owner-ref=true"
+    waitComplete:
+      - type: "pod"
+        selector: "app.kubernetes.io/name=webhook"
+        namespace: "cert-manager"
+      - type: "pod"
+        selector: "app.kubernetes.io/name=cainjector"
+        namespace: "cert-manager"
+      - type: "pod"
+        selector: "app.kubernetes.io/name=cert-manager"
+        namespace: "cert-manager"
+
+  - id: namespace-tekton
+    needs: linkerd
+    type: namespace
+    namespace: tekton
+    values:
+      - name: linkerd.io/inject
+        value: enabled
+        type: annotation
+
+  - id: tekton
+    needs: namespace-tekton
+    type: yaml
+    source:
+      path: assets/installer/pipeline-v0.28.0.yaml
+    waitComplete:
+      - type: "pod"
+        selector: "app=tekton-pipelines-webhook"
+        namespace: tekton-pipelines
+      - type: "crd"
+        selector: "clustertasks.tekton.dev"
+      - type: "crd"
+        selector: "conditions.tekton.dev"
+
+  - id: namespace-epinio
+    needs: linkerd
+    type: namespace
+    namespace: epinio
+    values:
+      - name: linkerd.io/inject
+        value: enabled
+        type: annotation
+
+  - id: epinio
+    needs: namespace-epinio
+    namespace: epinio
+    type: helm
+    source:
+      name: epinio
+      path: helm/chart/epinio/
```

Comment:
Given we want to move this helm chart to the helm chart repository, where is this manifest supposed to get it from? Chicken egg problem again? What if a PR on this repository needs a version of the helm chart which is on a branch on the helm chart repository (from another PR there)? We can introduce some CI commands to clone the helm chart from the other repository to some gitignored location here. But that means, that code should checkout a specific commit from the helm chart repository. And that should be removed before we merge. This is too complex. Any ideas @manno @olblak @andreas-kupries ?

Reply:
Let's handle this on a separate PR